sqlglot/dialects/dialect.py

```diff
@@ -318,6 +318,9 @@ class Dialect(metaclass=_Dialect):
     UNICODE_START: t.Optional[str] = None
     UNICODE_END: t.Optional[str] = None
 
+    # Separator of COPY statement parameters
+    COPY_PARAMS_SEP: t.Optional[TokenType] = None
```

Comment:
We should make this be `COMMA` by default

Reply:
Actually, if the only separators are comma and space, we should rename this to something like `COPY_PARAMS_ARE_CSV`